src/main/java/appeng/tile/misc/TileCharger.java

```diff
@@ -292,4 +235,71 @@ public void activate( final EntityPlayer player )
 			Platform.spawnDrops( this.worldObj, this.pos.offset( this.getForward() ), drops );
 		}
 	}
+
+	@Override
+	public TickingRequest getTickingRequest( IGridNode node )
+	{
+		return new TickingRequest( TickRates.Charger.getMin(), TickRates.Charger.getMin(), false, true );
+	}
+
+	@Override
+	public TickRateModulation tickingRequest( IGridNode node, int TicksSinceLastCall )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00117)